netlify.toml

```
@@ -1,4 +1,14 @@
 [build]
   base = "."
   command = "./utility_scripts/build/build_with_metrics.sh"
-  publish = "dist"
\ No newline at end of file
+  publish = "dist"
+
+[[redirects]]
+  from = "/taps/*"
+  to = "/extractors/:splat"
+  force = true
+
+[[redirects]]
+  from = "/targets/*"
+  to = "/loaders/:splat"
+  force = true
```

I'm not 100% sure about this, because we may have had some changes over time... but I think the last part of the endpoint name may need to be prefixed with `tap-` or `target-`.

```suggestion
[[redirects]]
  from = "/taps/*"
  to = "/extractors/tap-:splat"
  force = true

[[redirects]]
  from = "/targets/*"
  to = "/loaders/target-:splat"
  force = true
```

cc @pnadolny13 who may have more info

---

I see now, makes sense to me